capture-sdk/sdk/src/main/res/layout-sw600dp-land/gc_fragment_camera.xml

```diff
@@ -235,7 +235,7 @@
             android:scaleType="fitXY"
             android:src="@drawable/gc_camera_frame"
             app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintDimensionRatio="W, 1.414:1"
```

Comment:
[Clarification] Should `W` be removed also from the portrait tablet layout? Can you check please? Thank you!

Reply:
Sorry, my mistake. Please ignore my comment.